code_samples/simple_language/src/main/java/com/intellij/sdk/language/SimpleCompletionContributor.java

```diff
@@ -0,0 +1,28 @@
+// Copyright 2000-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
+
+package com.intellij.sdk.language;
+
+import com.intellij.codeInsight.completion.*;
+import com.intellij.codeInsight.lookup.LookupElementBuilder;
+import com.intellij.patterns.PlatformPatterns;
+import com.intellij.util.ProcessingContext;
+import com.intellij.sdk.language.psi.SimpleTypes;
+import org.jetbrains.annotations.NotNull;
+
+public class SimpleCompletionContributor extends CompletionContributor {
+  public SimpleCompletionContributor() {
+    // Register the completion providers
```

Comment:
let's remove all these line comments

Reply:
I removed the comments...but let's keep in mind this _isn't_ production code.